### PR TITLE
Add dogepedia page

### PR DESCRIPTION
**Pull request by bellesea**
Hopefully this works? I created some dummy pages to test but I think after most content is added, more work probably needs to be done. 

Also, for the resources pages, what sort of files will be added?

Would love your feedback! Merry Christmas :)

### CODE REVIEW

**Comment by bellesea** [comment]
A preview:
https://user-images.githubusercontent.com/65808924/147368113-d0fd0599-1679-41f2-a0ac-6d4f67e04629.mov

**Comment by xanimo** [comment]
> A preview: https://user-images.githubusercontent.com/65808924/147368113-d0fd0599-1679-41f2-a0ac-6d4f67e04629.mov

much wow! looks lovely belle! let me test it out rly quick :3

File: config.toml [hunk 1]
```diff
@@ -44,7 +44,7 @@ disableHugoGeneratorInject = false
       unsafe = true
 
 [permalinks]
-  posts = "/posts/:year/:month/:title/"
+  posts = "/:title/"
```
Comment (xanimo) [inline config.toml:47]: rationale?

File: themes/hello-friend-ng/layouts/partials/footer.html [hunk 2]
```diff
@@ -1,6 +1,6 @@
 <footer>
     <div class="footer-container">
-        <img class="footer-image" src="/footer-map.png"/>
+        <!-- <img class="footer-image" src="/footer-map.png"/> -->
```
Comment (xanimo) [inline themes/hello-friend-ng/layouts/partials/footer.html:3]: can we separate this into another PR?

**Review by xanimo — CHANGES_REQUESTED** [review]
looks good, have a few changes to request. removal of .DS_Store files and then if we can make a new PR for the changes to footer to keep it more atomic? great job regardless!

File: config.toml (hunk 1 above)
Comment (bellesea) [reply config.toml:47]: I found that with /posts.... the permalink did not lead to the correct place but with just /:title/, it actually led to the Dogepedia pages

Code:

`                <a href="{{.Permalink}}">
                    <button class="post-button">Read more</button>
                </a>`

File: themes/hello-friend-ng/layouts/partials/footer.html (hunk 2 above)
Comment (bellesea) [reply themes/hello-friend-ng/layouts/partials/footer.html:3]: will do! thanks :)

**Review by xanimo — APPROVED** [review]
ACK
